### PR TITLE
[Model Runner V2] Support min-p sampling

### PR DESCRIPTION
**Pull request by WoosukKwon**
This PR implements min-p sampling. Unlike the existing implementation in V1, this PR avoids softmax by directly comparing logits.

### CODE REVIEW

File: vllm/v1/worker/gpu/sample/metadata.py [hunk 1]
```diff
@@ -115,6 +120,9 @@ def _expand_sampling_metadata_kernel(
         top_k = tl.load(top_k_ptr + req_idx)
         tl.store(expanded_top_k_ptr + start_idx + block, top_k, mask=mask)
 
+    min_p = tl.load(min_p_ptr + req_idx)
+    tl.store(expanded_min_p_ptr + start_idx + block, min_p, mask=mask)
+
```
[the remark this answers is not in the document]
Comment (WoosukKwon) [reply vllm/v1/worker/gpu/sample/metadata.py]: Good suggestion. Fixed.
